README.md

```diff
@@ -185,6 +185,34 @@ The server must be accessible using the `$HOME/id_rsa` SSH Key, and **root** is
 
 **Note:** Running Streisand against an existing server can be a destructive action! You will be potentially overwriting configuration files and must be certain that you are affecting the correct machine.
 
+### Noninteractive Deployment (Advanced) ###
+
+Alternative scripts and configuration file examples are provided for
+noninteractive deployment, in which all of the required information is passed
+on the command line or in a configuration file.
+
+Example configuration files are found under `global_vars/noninteractive`. Copy
+and edit the desired parameters, such as providing API tokens and other choices,
+and then run the appropriate script.
+
+To deploy a new Streisand server:
+
+      deploy/streisand-new-cloud-server \
```

Comment:
Can you add a `.sh` extension to the shell scripts in the `deploy/` directory? That's how CI knows to run shellcheck. I admit it's a little but ugly compared to a bare path but I think it's a fair trade off to get bash scripts linted easily.

These example invocations in the README will have to change as well as the file names.

Thanks!